nerfstudio/process_data/process_data_utils.py

```diff
@@ -157,6 +158,7 @@ def convert_video_to_images(
         spacing = num_frames // num_frames_target
         if spacing > 1:
             ffmpeg_cmd += f" -vf thumbnail={spacing},setpts=N/TB{crop_cmd} -r 1"
+            print("Number of frames to extract:", math.ceil(num_frames / spacing))
```

Comment:
Can you switch to `CONSOLE.print(`

Reply:
Done; I also switched the other print statement that reports total number of frames.